### DISCUSSION TITLE
Can we strip large python binaries?

### DISCUSSION THREAD

**Discussion by hemnstill** [indygreg/python-build-standalone#160] [Q&A] [answered]
For example, `python/install/bin/python3.11` from https://github.com/indygreg/python-build-standalone/releases/download/20230116/cpython-3.11.1+20230116-x86_64-unknown-linux-musl-noopt-full.tar.zst 

size before strip is 34,7 MB, after strip is 18,9 MB

**Comment by indygreg** [comment, marked as answer]
This project already uploads a few hundred release artifacts. Having stripped versions would compound the number of releases.

My knee jerk is if you want stripped binaries, download the official tarballs and strip the binaries.

I'm not opposed to putting a script in the release tarballs that makes this a turn-key process.

**Comment by bulletmark** [comment]
To follow up on this, wouldn't it be reasonable that the binaries in the `install_only` tar file at least be stripped?

**Reply by sanmai-NL** [reply]
@indygreg I think you missed this comment.

**Reply by zanieb** [reply]
There are [install_only_stripped](https://github.com/astral-sh/python-build-standalone/releases/download/20250317/cpython-3.10.16+20250317-aarch64-unknown-linux-gnu-install_only_stripped.tar.gz) variants now.
